tools/src/dh5walk/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required (VERSION 3.12)
+project (HDF5_TOOLS_SRC_DH5WALK C)
+
+# --------------------------------------------------------------------
+# Add the dh5walk and test executables
+# --------------------------------------------------------------------
+if (NOT ONLY_SHARED_LIBS)
+  add_executable (dh5walk ${HDF5_TOOLS_SRC_DH5WALK_SOURCE_DIR}/dh5walk.c)
+  add_custom_command(
+	OUTPUT ${HDF5_TOOLS_DIR}/test/demo_destfiles.test
+	COMMAND bash -c ${HDF5_TOOLS_SRC_DH5WALK_SOURCE_DIR}/copy_demo_files.sh
```

Comment:
Can we convert this script to use the cmake copy method? Search for "HDFTEST_COPY_FILE" in cmaketests.cmake files. This will not execute on windows, and would need to be guarded against (which we do for the TEST_SHELL_SCRIPTS)